libs/explored-types/src/types.ts

```diff
@@ -104,14 +105,148 @@ export type TxpoolTransactions = {
   v2transactions: V2Transaction[]
 }
 
+/**
+ * HostSettings are the settings and prices used when interacting with a host.
+ */
+export type HostSettings = {
```

Comment:
I think we have the same types already in `renterd-types`: https://github.com/SiaFoundation/web/blob/704d5a629f011abc89ba3c6b61004d01a46af22b/libs/renterd-types/src/types.ts#L23-L84

Could you move them into `types` and import then into `renterd-types` and `explored-types`.

Reply:
Sure thing. Handled in a down stack PR.